appendices/versions.txt

```diff
@@ -188,37 +188,71 @@ number of key extensions into the core API:
 
 include::{generated}/meta/promoted_extensions_VK_VERSION_1_1.txt[]
 
-The only changes to the functionality added by these extensions were to
-`VK_KHR_shader_draw_parameters`, which had the
-<<features-shaderDrawParameters, pname:shaderDrawParameters>> feature bit
-added to determine support in the core API, and
-<<features-variablePointersStorageBuffer,
-pname:variablePointersStorageBuffer>> from `VK_KHR_variable_pointers` was
-made optional.
+All differences in behavior between these extensions and the corresponding
+Vulkan 1.1 functionality are summarized below.
 
-[[versions-1.1-new-features]]
-Additionally, Vulkan 1.1 added support for <<shaders-group-operations, group
-operations>> with <<shaders-scope-subgroup, subgroup scope>>,
-<<memory-protected-memory, protected memory>>, and a new command to
-<<vkEnumerateInstanceVersion, enumerate the instance version>>.
+=== Differences relative to `VK_KHR_16bit_storage`
+
+If the `<<VK_KHR_16bit_storage>>` extension is not supported, support for
+the <<features-storageBuffer16BitAccess, pname:storageBuffer16BitAccess>>
+feature is optional.
+Support for this feature is defined by
+slink:VkPhysicalDevice16BitStorageFeatures::pname:storageBuffer16BitAccess
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:storageBuffer16BitAccess
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
 === Differences relative to `VK_KHR_sampler_ycbcr_conversion`
 
 If the `<<VK_KHR_sampler_ycbcr_conversion>>` extension is not supported,
-support for the <<features-samplerYcbcrConversion,samplerYcbcrConversion>>
+support for the <<features-samplerYcbcrConversion, pname:samplerYcbcrConversion>>
 feature is optional.
 Support for this feature is defined by
-slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion when
-queried via flink:vkGetPhysicalDeviceFeatures2.
+slink:VkPhysicalDeviceSamplerYcbcrConversionFeatures::pname:samplerYcbcrConversion
+ifdef::VK_VERSION_1_2[]
+or slink:VkPhysicalDeviceVulkan11Features::pname:samplerYcbcrConversion
+endif::VK_VERSION_1_2[]
+when queried via flink:vkGetPhysicalDeviceFeatures2.
 
-=== Differences relative to `VK_KHR_16bit_storage`
+=== Differences relative to `VK_KHR_shader_draw_parameters`
 
-If the `<<VK_KHR_16bit_storage>>` extension is not supported, support for
-the <<features-storageBuffer16BitAccess,storageBuffer16BitAccess>> feature
-is optional.
+If the `<<VK_KHR_shader_draw_parameters>>` extension is not supported,
+support for the <<features-shaderDrawParameters, pname:shaderDrawParameters>>
```

Comment:
`VK_KHR_shader_draw_parameters` should match more of the wording of `VK_KHR_draw_indirect_count` as both didn't have the feature bit prior to being promoted

I suggest

> If the VK_KHR_shader_draw_parameters extension is not supported, shader draw parameters are optional. Support for this feature is defined by  ...

Reply:
I believe it already is. It is copied from `VK_EXT_descriptor_indexing`, I think.
What is the exact diff here that you see?

Reply:
There was no `shaderDrawParameters` before 1.1, Prior if you had the extension enabled, it then allowed you to have shader draw parameters, therefore, like `VK_KHR_draw_indirect_count` which also didn't have a feature bit, falls in the same group

The difference is "there was no feature bit in the extension and it was added with the promotion" it is a subtle detail, but there isn't much more to it than that

Reply:
Yea I get that. I mean, where do you see the difference between my wording and the `VK_KHR_draw_indirect_count`?

Reply:
We have

> If the VK_KHR_draw_indirect_count extension is not supported, support for the entry points vkCmdDrawIndirectCount and vkCmdDrawIndexedIndirectCount is optional.

not 

> If the VK_KHR_draw_indirect_count extension is not supported, support for the <<features-drawIndirectCount, pname:drawIndirectCount>> is optional.

because `drawIndirectCount` was not there prior to promotion, same with `shaderDrawParameters`

Reply:
The `VK_EXT_descriptor_indexing` (which also didn't have a feature flag) has it the way I do. Probably needs some consistency changes later. Though what you suggest might be unsustainable for some of the extensions, as it would require naming ~20 entrypoints, enums, and pnext structs that it added.

For now I am just gonna do it then, as it seems it only adds `SPV_KHR_shader_draw_parameters`

Reply:
done

Reply:
I agree `VK_EXT_descriptor_indexing` should be fixed (I can take care of that)